ironfish/src/telemetry/telemetry.ts

```diff
@@ -35,13 +36,15 @@ export class Telemetry {
   constructor(options: {
     chain: Blockchain
     workerPool: WorkerPool
+    graffiti?: string
     logger?: Logger
     metrics?: MetricsMonitor
     defaultFields?: Field[]
     defaultTags?: Tag[]
   }) {
     this.chain = options.chain
     this.workerPool = options.workerPool
+    this.graffiti = options.graffiti ?? ''
```

Comment:
I think most people will have this enabled, but we could leave this as optional and conditionally check for assignment in the API.

Reply:
I thought about that, but I didn't see any examples of optional params in the worker pool message serializers so wasn't sure if that was going to be a lot of extra work for temporary code

Reply:
Makes sense. [This](https://github.com/iron-fish/ironfish/blob/91fe59d1bf1981ed224ef5ea4465dd35ee386ba8/ironfish/src/network/messages/disconnecting.ts#L40-L66) is how I am doing it for network serialization if you'd want to do that here - either way.

Reply:
Ah perfect, thanks, I'll follow that model

Reply:
This is _kind_ of moot now that we pass in the config here instead of graffiti, but just to make sure: I had this technically kind of wrong, since `config.get('blockGraffiti')` will always return a string, `''` is the default value